lib/Myriad.pm

```diff
@@ -306,6 +308,22 @@ method redis () {
     $redis
 }
 
+=head2 perl_transport
```

Comment:
This feels like a step in the wrong direction - we should be making things transport-agnostic, not bringing in specific named variables for different types of transport!

Reply:
As discussed internally, this change will be done in another PR after having the concept discussed.